examples/chip-tool/commands/common/CHIPCommand.h

```diff
@@ -57,8 +57,8 @@ class CHIPCommand : public Command
     using PeerId                 = ::chip::PeerId;
     using PeerAddress            = ::chip::Transport::PeerAddress;
 
-    static constexpr uint16_t kMaxGroupsPerFabric    = 5;
-    static constexpr uint16_t kMaxGroupKeysPerFabric = 8;
+    static constexpr uint16_t kMaxGroupsPerFabric    = 50;
```

Comment:
isn't there a Config define that this should be linked with? 
I believe `CHIP_CONFIG_MAX_GROUPS_PER_FABRIC`. It isn't used in chip-tool?

Reply:
The constructor override the default settings.